pkg/app/pipectl/cmd/appconfig/rename.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package appconfig
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipecd/pkg/app/server/service/apiservice"
+	"github.com/pipe-cd/pipecd/pkg/cli"
+	"github.com/pipe-cd/pipecd/pkg/model"
+)
+
+type rename struct {
+	root *command
+
+	repoRootPath         string
+	repoID               string
+	envName              string
+	before               string
+	after                string
+	updateAtLocal        bool
+	updateOnControlPlane bool
+	stdout               io.Writer
+}
+
+func newRenameCommand(root *command) *cobra.Command {
+	c := &rename{
+		root:   root,
+		stdout: os.Stdout,
+	}
+	cmd := &cobra.Command{
+		Use:   "rename",
+		Short: "Finds all applications that has the given configuration file name to change.",
+		RunE:  cli.WithContext(c.run),
+	}
+
+	cmd.Flags().StringVar(&c.repoRootPath, "repo-root-path", c.repoRootPath, "The absolute path to the root directory of the Git repository.")
+	cmd.Flags().StringVar(&c.repoID, "repo-id", c.repoID, "The repository ID that is being registered in Piped config.")
+	cmd.Flags().StringVar(&c.envName, "env-name", c.envName, "The environment name.")
+	cmd.Flags().StringVar(&c.before, "before", c.before, "The current name of configuration file.")
+	cmd.Flags().StringVar(&c.after, "after", c.after, "The new name of configuration file.")
+	cmd.Flags().BoolVar(&c.updateAtLocal, "update-at-local", c.updateAtLocal, "Whether to rename files in Git locally.")
```

Comment:
should be `updateOnLocal` to unify with the `updateOnControlPlane`?

Reply:
@khanhtc1202 Applied it!